src/main/java/nextstep/subway/favorite/application/FavoriteService.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.subway.favorite.application;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import nextstep.subway.auth.domain.LoginMember;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.domain.FavoriteRepository;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.member.application.MemberService;
+import nextstep.subway.member.domain.Member;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.Station;
+
+@Service
+@Transactional
+public class FavoriteService {
+	private final FavoriteRepository favoriteRepository;
+	private final StationService stationService;
+	private final MemberService memberService;
+
+	public FavoriteService(FavoriteRepository favoriteRepository, StationService stationService,
+		MemberService memberService) {
+		this.favoriteRepository = favoriteRepository;
+		this.stationService = stationService;
+		this.memberService = memberService;
+	}
+
+	public FavoriteResponse createFavorite(LoginMember loginMember, FavoriteRequest request) {
+		Member member = memberService.findById(loginMember.getId());
```

Comment:
loginMember를 넘겨서 `getId()`만 호출하고 있는것으로 보이네요.

id만 넘기다가 이후에 로그인 사용자의 추가 정보가 필요해지면 통채로 넘기거나 하면 어떨까요?

```suggestion
	public FavoriteResponse createFavorite(Long memberId, FavoriteRequest request) {
		Member member = memberService.findById(loginMember);
```

Reply:
서비스 계층의 로직을 최소화하는 측면에서 좋은 아이디어라고 생각됩니다!
반영하여 수정할께요.
커멘트 감사합니다!